packages/storybook-blocks/.eslintrc.js

```diff
@@ -39,7 +39,7 @@ const commonExtends = ['plugin:react/recommended', 'plugin:prettier/recommended'
 module.exports = {
   overrides: [
     {
-      files: ['*.jest.js', 'jest.init.js'],
+      files: ['*.test.js', 'jest.init.js'],
```

Comment:
same as before, what's the meaning of it not catching ts{x}

Reply:
Removed it as it has no effect